lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
I meant that `form_for ""` is probably illegal in rails terms but it looks like the gem's author uses a trick to centralize the custom form stuff, via a common form builder (for both `form_for` and `form_tag`).

I'll try to look into it later.